openmdao/core/vec_wrapper_complex_step.py

```diff
@@ -0,0 +1,298 @@
+""" VecWrapper 'wrapper' that is used for component-wise complex step."""
+
```

Comment:
would be interested in @naylor-b thoughts...   just from first read-thru, it seems like it could be done with just the couple self-contained functions added to VecWrapper... if so, that seems preferable to me